test/jdk/java/net/DatagramSocket/DatagramSocketMulticasting.java

```diff
@@ -48,6 +48,9 @@
 
 import jdk.test.lib.NetworkConfiguration;
 import jdk.test.lib.net.IPSupport;
+import jtreg.SkippedException;
```

Comment:
jtreg.SkippedWxception is probably no longer needed, since you are using `assumeTrue`?